### PR TITLE
Create Rule “establish-the-world/rule”

### PR DESCRIPTION
**Pull request by jontygardner**
Automatically generated by Netlify CMS

### CODE REVIEW

**Review by tomek-i — APPROVED** [review]
LGTM

**Comment by tiagov8** [comment]
> LGTM, however there seems to be an issue with the front-matter.

Hey @tomek-i 
Can you give more info?

File: rules/establish-the-world/rule.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+type: rule
+title: Do you establish the world for your audience?
+uri: establish-the-world
+authors:
+  - title: Jonty Gardner
+  - url: https://ssw.com.au/people/jonty-gardner
+created: 2023-10-04T06:22:53.243Z
+guid: f1dac5ed-a422-4e19-bf72-f6c637ba0a23
+---
+## 🎥 Do you establish the world for your audience?
```
Comment (tiagov8) [inline rules/establish-the-world/rule.md:11]: Hey @jontygardner 

No reason to repeat the title here... 

1. Please delete this heading

File: rules/establish-the-world/rule.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+- **Professionalism:** Incorporating drone footage signals a commitment to high-quality production, which can leave a lasting positive impression on your audience.
+
+- **Visual Impact:** Drone shots can showcase epic landscapes and unique perspectives, captivating viewers and making your content visually engaging.
+
+- **Spatial Awareness:** By showing the environment from above, drone footage helps viewers understand the geographical context of the video's location.
+
+### Examples:
+
+::: greybox {
+   [youtube: 'https://youtu.be/7jOJ2QlfGUg']
```
Comment (tiagov8) [inline rules/establish-the-world/rule.md:28]: Hey @jontygardner 

1. Please check how to add videos on https://ssw.com.au/rules/rule/#8-videos and fix the Markdown in this new rule

File: rules/establish-the-world/rule.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+::: bad Figure: Bad Example
+The video starts abruptly without establishing the setting, leaving viewers confused about where the content is taking place.
+:::
+
+::: greybox {
+   [youtube: 'https://youtu.be/uFLnYo_z6Pw']
+} :::
+
+::: good Figure: Good Example
```
Comment (tiagov8) [inline rules/establish-the-world/rule.md:39]: @jontygardner 

1. Please fix to be like the below
2. Do the same for the bad example (attention to small case "example")

```
::: good 
Figure: Good example - The video begins (after the hook) with stunning drone footage of the scenic location, setting the stage for the content and immersing the audience in the world of the video.
:::
```

File: rules/establish-the-world/rule.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+created: 2023-10-04T06:22:53.243Z
+guid: f1dac5ed-a422-4e19-bf72-f6c637ba0a23
+---
+## 🎥 Do you establish the world for your audience?
+
+When creating video content, it's crucial to start by setting up the world and the location where your content is taking place. This initial impression can significantly impact how your audience engages with your video. One effective way to achieve this is through the use of drone footage.
+
+Drones have become increasingly popular and offer the ability to capture breathtaking landscape shots and viewpoints that were once impossible to achieve a decade ago. Investing in drone footage not only adds a touch of professionalism but also elevates the overall production quality of your video.
+
+### Why Establishing the World Matters:
```
Comment (tiagov8) [inline rules/establish-the-world/rule.md:17]: @jontygardner 

1. Please remove colon from headings

File: rules/establish-the-world/rule.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+
+- **Professionalism:** Incorporating drone footage signals a commitment to high-quality production, which can leave a lasting positive impression on your audience.
```
Comment (tiagov8) [inline rules/establish-the-world/rule.md:19]: Hey @jontygardner 

1. Can you follow https://www.ssw.com.au/rules/avoid-full-stops-in-bullet-point-lists here?

**Comment by tomek-i** [comment]
> > LGTM, however there seems to be an issue with the front-matter.
> 
> Hey @tomek-i Can you give more info?

@tiagov8 
Complaining about invalid authors. I just picked a random other rule https://github.com/SSWConsulting/SSW.Rules.Content/blob/main/rules/acceptance-criteria/rule.md?plain=1
and I don't see a difference but the check on this item is failing.

image

File: rules/establish-the-world/rule.md (hunk 1 above)
Comment (jontygardner) [reply rules/establish-the-world/rule.md]: Done

**Comment by tiagov8** [comment]
Hey @jontygardner 

regarding the issue @tomek-i found above

1. Please remove the dash from URL from

```
authors:
  - title: Jonty Gardner
  - url: https://ssw.com.au/people/jonty-gardner
```

to

```
authors:
  - title: Jonty Gardner
    url: https://ssw.com.au/people/jonty-gardner
```

**Review by jontygardner — COMMENTED** [review]
Changes made

**Pull request merged**